Significant-Gravitas/Auto-GPT
# 'browse_website' command will stuck in create WebDriver Chrome instance, when exporting the 'https_proxy' env var in terminal.

### Background
OpenAI's API blocks calls from certain countries, so they have to use proxies to access it. Using proxies in the terminal is typically done by exporting the environment variable 'https_proxy'. However, after exporting this variable, `webdriver.Chrome` get blocked and the program gets stuck.

### Changes
Before creating a Chrome instance, remove the 'https_proxy' environment variable. After creating the instance, restore the environment variable so that the OpenAI API can be called normally.


### Test Plan
1. Export environment variable `export https_ proxy= http://127.0.0.1:1080`
2. Run the Auto-GPT program `python - m autogpt -- gpt3only`
3. Set a simple task: Open a page using a browser, such as: https://platform.openai.com/docs/introduction
4. Verify whether the task can be completed normally

### PR Quality Checklist
- [ x ] My pull request is atomic and focuses on a single change.
- [ x ] I have thoroughly tested my changes with multiple different prompts.
- [ x ] I have considered potential risks and mitigations for my changes.
- [ x ] I have documented my changes clearly and comprehensively.
- [ x ] I have not snuck in any "extra" small tweaks changes
